src/plotly/plotly_types.nim

```diff
@@ -162,8 +178,12 @@ type
     case `type`*: PlotType
     of HeatMap, HeatMapGL:
       colormap*: ColorMap
+      customCmap*: CustomColorMap
```

Comment:
how about `customColorMap` instead of `customCmap`?

Reply:
Yeah, you're right. Only capitalized the `C` though, to keep consistend with `colormap` and `colorscale` fields.